Fixed upgrade tower dialog bug

Megafont opened the pull request:
+ Fixed collider on the neutrophil tower. It was way to big, causing the dialog to appear even if you clicked well outside the tower
+ Added cost to the tower selector info popup since that was mentioned in testing

Megafont committed:
Fixed upgrade tower dialog bug

+ Fixed collider on the neutrophil tower. It was way to big, causing the dialog to appear even if you clicked well outside the tower
+ Added cost to the tower selector info popup since that was mentioned in testing

wolfyman411 commented:
Macrophage upgrade area is too large, and I can't upgrade placed lymphocytes. Everything else looks fine.

wolfyman411 reviewed (changes requested):
Macrophage upgrade area is too large, and I can't upgrade placed lymphocytes. Everything else looks fine.

Megafont committed:
Fixed issue with macrophage/neutrophil towers

+ It was collider issues again. I removed the collider child object on both offenders
+ I also disabled the Queries Hit Triggers option in ProjectSettings->Physics so the large colliders will be ignored by the mouse now

Megafont committed:
Merge branch 'main' into Megafont_FixUpdateDialogBug

Megafont commented:
The problem was colliders again. I removed the child collider object from both the macrophage and neutrophil towers
I also disabled ProjectSettings->Physics->QueriesHitTriggers option so mouse clicks will no ignore the larger colliders that are set as triggers

wolfyman411 committed:
Fixed some issues
